resources/views/pages-legacy/achievementInfo.blade.php

```diff
@@ -470,18 +470,31 @@ function ResetProgress() {
     if ($unlocks->isEmpty()) {
         echo "Nobody yet! Will you be the first?!<br>";
     } else {
+        $userWinners = $unlocks->pluck('User')->filter()->unique()->values()->toArray();
```

Comment:
Why is `->filter()->unique()->values()` necessary? Can't you just `pluck('User')->toArray()`?

Reply:
Taking a 2nd look at `getAchievementUnlocksData()`, I way overcomplicated this. We don't need this much defensiveness given the implementation details of that function. Simplified in latest to `pluck('User')->toArray()`.